README.org

```diff
@@ -2,4 +2,4 @@
 Procedurally generated sunrises and sunsets
 
 * To start
-=(figwheel-up)=
```

Comment:
I think we should leave both of these.  I still start it by using `dev.repl> (figwheel-up)` after invoking `cider-jack-in` in a clojure file in the project.